cartography/data/jobs/cleanup/aws_import_codeguru_associations_cleanup.json

```diff
@@ -0,0 +1,11 @@
+{
+    "statements": [
+      {
+        "query": "MATCH (:AWSAccount{id: {AWS_ID}})-[:RESOURCE]->(cga:CodeguruAssociation) WHERE cga.lastupdated <> {UPDATE_TAG} WITH cga LIMIT {LIMIT_SIZE} DETACH DELETE (cga) return COUNT(*) as TotalCompleted",
```

Comment:
It seems `return COUNT(*) as TotalCompleted` is no longer needed in the analysis jobs (cartography handles this automatically now).